api/models/contact_fields_mapping.py

```diff
@@ -0,0 +1,32 @@
+from api import db
+from flask_sqlalchemy import BaseQuery
+
+
+class ContactFieldsMappingQuery(BaseQuery):
+    def get_by_field_name(self, field_name, field_value):
+        try:
+            return self.filter(
+                ContactFieldsMapping.field_name == field_name,
+                ContactFieldsMapping.expected_field_value == field_value,
+            ).all()
+        except Exception as e:
+            print(f"Exception occurred: {e}")
+
+    def get_by_group_name(self, contact_groups):
+        try:
+            return self.filter(ContactFieldsMapping.field_name == contact_groups).all()
+        except Exception as e:
+            print(f"Exception occurred: {e}")
+
+
+class ContactFieldsMapping(db.Model):
+    query_class = ContactFieldsMappingQuery
+    __tablename__ = "contact_fields_mapping"
+    id = db.Column(db.Integer, primary_key=True)
+    field_type = db.Column(db.String(255))
+    field_name = db.Column(db.String(255))
+    mapped_table_name = db.Column(db.String(255))
+    mapped_table_column_name = db.Column(db.String(255))
+    mapped_table_column_value = db.Column(db.String(255))
+    expected_field_data_type = db.Column(db.String(255), nullable=True)
+    expected_field_value = db.Column(db.String(255), nullable=True)
```

Comment:
What is the use of these 3 columns?

Reply:
- **mapped_table_column_value:** Value to be added in the mapped table column.
- **expected_field_data_type:** Contains the data type of the expected field value.
- **expected_field_value**: Contains the expected value of the contact field name and is compared with the field value we are getting.

Reply:
Got it